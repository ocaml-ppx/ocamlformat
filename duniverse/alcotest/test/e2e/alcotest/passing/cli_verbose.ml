(** Ensures that the `--verbose` flag is passed from the CLI. *)

let () =
  let open Alcotest in
  run ~verbose:false (* CLI flag should take priority over this option *)
    "cli_verbose"
    [
      ( "alpha",
        [
          test_case "0" `Quick (fun () ->
              Format.printf "SHOULD APPEAR IN TEST OUTPUT\n");
        ] );
    ]
