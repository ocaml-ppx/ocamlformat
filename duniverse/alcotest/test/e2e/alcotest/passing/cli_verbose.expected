Testing `cli_verbose'.
This run has ID `<uuid>'.

SHOULD APPEAR IN TEST OUTPUT
  [OK]          alpha          0   0.

Test Successful in <test-duration>s. 1 test run.
